src/sdk-installer.ts

```diff
@@ -19,11 +19,11 @@ export async function installAndroidSdk(apiLevel: string, target: string, arch:
     const isOnMac = process.platform === 'darwin';
     const isArm = process.arch === 'arm64';
 
-    if (!isOnMac) {
-      await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/platform-tools -R`);
-      await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/cmdline-tools/latest -R`);
-      await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/build-tools/${BUILD_TOOLS_VERSION} -R`);
-    }
+    // if (!isOnMac) {
+    //   await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/platform-tools -R`);
+    //   await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/cmdline-tools/latest -R`);
+    //   await exec.exec(`sh -c \\"sudo chown $USER:$USER ${process.env.ANDROID_HOME}/build-tools/${BUILD_TOOLS_VERSION} -R`);
+    // }
```

Comment:
not sure why the chown was there in the first place, but at least to have it not fail - #412 works well